services/images/local.go

```diff
@@ -159,7 +175,12 @@ func (l *local) Update(ctx context.Context, req *imagesapi.UpdateImageRequest, _
 
 func (l *local) Delete(ctx context.Context, req *imagesapi.DeleteImageRequest, _ ...grpc.CallOption) (*ptypes.Empty, error) {
 	log.G(ctx).WithField("name", req.Name).Debugf("delete image")
+	span := trace.SpanFromContext(ctx)
+	_, imageDeleteSpan := (span.Tracer().Start(ctx, "imageDeleteSpan", trace.WithAttributes(attribute.String("imageName", req.Name))))
```

Comment:
This stuff gets messy fast because we end up wanting to log to the logger as well as to the span.